assets/unsplash/README.md

```diff
@@ -0,0 +1,17 @@
+Files in this directory are covered by the [Unsplash license](https://unsplash.com/license).
+
+assorted-color-textiles.webp
```

Comment:
TODO: Replace this image with the midjourney-generated embroidered world map. Keep the fade-out effect.